openaq_api/openaq_api/v3/routers/manufacturers.py

```diff
@@ -78,7 +83,27 @@ async def manufacturers_get(
 
 async def fetch_manufacturers(query, db):
     query_builder = QueryBuilder(query)
-    sql = f"""
-    """
+    sql = f"""   
+        SELECT 
+            e.entities_id AS id
+            , e.full_name AS "name"
+            , ARRAY_AGG(DISTINCT (JSON_BUILD_OBJECT('id', i.instruments_id, 'name', i.label))::jsonb) AS instruments
```

Comment:
`JSON_BUILD_OBJECT` to lowercase `json_build_object` and we dont need to cast to jsonb, the plain json should be fine